content/kubeone/master/tutorials/creating_clusters/_index.en.md

```diff
@@ -655,10 +655,10 @@ In the following table, you can find a list of supported Kubernetes version
 for latest KubeOne versions (you can run `kubeone version` to find the version
 that you're running).
 
-| KubeOne version | 1.20       | 1.19 | 1.18 | 1.17 |
-| --------------- | ---------- | ---- | ---- | ---- |
-| v1.2+         | ✓ | ✓    | ✓    | -   |
-| v1.0+         | - | ✓    | ✓    | ✓\*\*   |
+| KubeOne version | 1.21       | 1.20       | 1.19 | 1.18 | 1.17 |
```

Comment:
Similar to the comment above -- can you update this table in the docs for v1.3 (this document doesn't exist in the v1.2 docs)?